### PR TITLE
contributors

### PR DESCRIPTION
**Pull request by egolol**
pictures and readme

### CODE REVIEW

**Review by giovannivolpe — CHANGES_REQUESTED** [review]
can you make the page more in this format?
http://braph.org/braph-1-0/contact-us/

Make the list of names and affiliations on the left, with affiliations and emails.
Also, ensure that the photos are not stretched, and all with the same size.

File: contributors/readme.md [hunk 1]
```diff
@@ -1 +1,14 @@
-
+# BRAPH 2.0 — Contributors
+|   | Information |
+| ------------- | ------------- |
+| ![Giovanni Volpe](figures/giovanni_volpe.png)  | Dr. Giovanni Volpe <br> Department of Physics, University of Gothenburg, <br>  Gothenburg, Sweden <br> giovanni.volpe@physics.gu.se  |
+| ![Joanna Pereira](figures/joana_pereira.png)  | Dr. Joanna B. Pereira <br> Department of Neurobiology, Care Sciences and Society <br> Karolinska Institutet, Stockholm, Sweden <br> joana.pereira@ki.se |
+| ![Emiliano Gomez](figures/emiliano_gomez.png) | Emiliano Gomez Ruiz <br> Department of Physics, University of Gothenburg, <br>  Gothenburg, Sweden <br> emiliano.gomez@physics.gu.se |
+| ![Anna Canal](figures/anna_canal.png) | Anna Canal García <br> Department of Neurobiology, Care Sciences and Society <br> Karolinska Institutet, Stockholm, Sweden <br> anna.canal@ki.se |
```
Comment (bzufiriaki) [inline contributors/readme.md]: Now we are in the "Department of Clinical Neuroscience, Division of Neuro"

File: contributors/readme.md [hunk 2]
```diff
@@ -1 +1,14 @@
...
+| ![Yu-Wei Chang](figures/yuwei_chang.png)| Yu-Wei Chang <br> Department of Physics, University of Gothenburg, <br>  Gothenburg, Sweden <br> yuwei.chang@physics.gu.se |
+| ![Mite Mijalkov](figures/mite_mijalkov.png) | Dr. Mite Mijalkov <br> Department of Neurobiology, Care Sciences and Society <br> Karolinska Institutet, Stockholm, Sweden <br> mite.mijalkov@ki.se |
+| ![Eric Westman](figures/eric_westman.png) | Dr. Eric Westman <br> Department of Neurobiology, Care Sciences and Society <br> Karolinska Institutet, Stockholm, Sweden <br> eric.westman@ki.se |
+| ![Daniel Vereb](figures/daniel_vereb.png) | Dr. Daniel Vereb <br> Department of Neurobiology, Care Sciences and Society <br> Karolinska Institutet, Stockholm, Sweden <br> daniel.vereb@ki.se |
+| ![Hang Zhao](figures/hang_zhao.png) | Hang Zhao <br> Department of Physics, University of Gothenburg, <br>  Gothenburg, Sweden <br> hang.zhao@physics.gu.se |
+| ![Blanca Zufiria](figures/blanca_zufiria.png) | Blanca Zufiria <br> Department of Neurobiology, Care Sciences and Society <br> Karolinska Institutet, Stockholm, Sweden <br> blanca.zufiria@ki.se |
```
Comment (bzufiriaki) [inline contributors/readme.md]: blanca.zufiria.gerboles@ki.se

File: contributors/readme.md (hunk 1 above)
Comment (annacanal) [inline contributors/readme.md]: Anna Canal **Garcia** <br> Department of Clinical Neuroscience <br> Karolinska Institutet, Stockholm, Sweden <br> **anna.canal.garcia@ki.se**

File: contributors/readme.md [hunk 3]
```diff
@@ -1 +1,14 @@
-
+# BRAPH 2.0 — Contributors
+|   | Information |
+| ------------- | ------------- |
+| ![Giovanni Volpe](figures/giovanni_volpe.png)  | Dr. Giovanni Volpe <br> Department of Physics, University of Gothenburg, <br>  Gothenburg, Sweden <br> giovanni.volpe@physics.gu.se  |
+| ![Joanna Pereira](figures/joana_pereira.png)  | Dr. Joanna B. Pereira <br> Department of Neurobiology, Care Sciences and Society <br> Karolinska Institutet, Stockholm, Sweden <br> joana.pereira@ki.se |
```
Comment (annacanal) [inline contributors/readme.md]: Department of Clinical Neuroscience

File: contributors/readme.md (hunk 2 above)
Comment (annacanal) [inline contributors/readme.md]: Department of Clinical Neuroscience

File: contributors/readme.md [hunk 4]
```diff
@@ -1 +1,14 @@
...
+| ![Emiliano Gomez](figures/emiliano_gomez.png) | Emiliano Gomez Ruiz <br> Department of Physics, University of Gothenburg, <br>  Gothenburg, Sweden <br> emiliano.gomez@physics.gu.se |
+| ![Anna Canal](figures/anna_canal.png) | Anna Canal García <br> Department of Neurobiology, Care Sciences and Society <br> Karolinska Institutet, Stockholm, Sweden <br> anna.canal@ki.se |
+| ![Yu-Wei Chang](figures/yuwei_chang.png)| Yu-Wei Chang <br> Department of Physics, University of Gothenburg, <br>  Gothenburg, Sweden <br> yuwei.chang@physics.gu.se |
+| ![Mite Mijalkov](figures/mite_mijalkov.png) | Dr. Mite Mijalkov <br> Department of Neurobiology, Care Sciences and Society <br> Karolinska Institutet, Stockholm, Sweden <br> mite.mijalkov@ki.se |
+| ![Eric Westman](figures/eric_westman.png) | Dr. Eric Westman <br> Department of Neurobiology, Care Sciences and Society <br> Karolinska Institutet, Stockholm, Sweden <br> eric.westman@ki.se |
+| ![Daniel Vereb](figures/daniel_vereb.png) | Dr. Daniel Vereb <br> Department of Neurobiology, Care Sciences and Society <br> Karolinska Institutet, Stockholm, Sweden <br> daniel.vereb@ki.se |
```
Comment (annacanal) [inline contributors/readme.md]: Department of Clinical Neuroscience

File: contributors/readme.md [hunk 5]
```diff
@@ -1 +1,14 @@
...
+|   | Information |
+| ------------- | ------------- |
+| ![Giovanni Volpe](figures/giovanni_volpe.png)  | Dr. Giovanni Volpe <br> Department of Physics, University of Gothenburg, <br>  Gothenburg, Sweden <br> giovanni.volpe@physics.gu.se  |
+| ![Joanna Pereira](figures/joana_pereira.png)  | Dr. Joanna B. Pereira <br> Department of Neurobiology, Care Sciences and Society <br> Karolinska Institutet, Stockholm, Sweden <br> joana.pereira@ki.se |
+| ![Emiliano Gomez](figures/emiliano_gomez.png) | Emiliano Gomez Ruiz <br> Department of Physics, University of Gothenburg, <br>  Gothenburg, Sweden <br> emiliano.gomez@physics.gu.se |
+| ![Anna Canal](figures/anna_canal.png) | Anna Canal García <br> Department of Neurobiology, Care Sciences and Society <br> Karolinska Institutet, Stockholm, Sweden <br> anna.canal@ki.se |
+| ![Yu-Wei Chang](figures/yuwei_chang.png)| Yu-Wei Chang <br> Department of Physics, University of Gothenburg, <br>  Gothenburg, Sweden <br> yuwei.chang@physics.gu.se |
+| ![Mite Mijalkov](figures/mite_mijalkov.png) | Dr. Mite Mijalkov <br> Department of Neurobiology, Care Sciences and Society <br> Karolinska Institutet, Stockholm, Sweden <br> mite.mijalkov@ki.se |
```
Comment (annacanal) [inline contributors/readme.md]: Department of Clinical Neuroscience

**Pull request merged**